tests/suites/test_suite_pk.function

```diff
@@ -716,7 +716,7 @@ exit:
 }
 /* END_CASE */
 
-/* BEGIN_CASE depends_on:MBEDTLS_PK_CAN_ECDSA_VERIFY */
+/* BEGIN_CASE depends_on:MBEDTLS_PK_CAN_ECDSA_VERIFY:MBEDTLS_ECP_LIGHT */
```

Comment:
This is correct, but I'm wondering it we could do better. Any way we should change the test code so that it doesn't require `ECP_LIGHT` any more?